docker-compose.yml

```diff
@@ -20,8 +20,8 @@ services:
       - "5433:5432" # order is host:container
     logging:
       driver: "none"
-    volumes:
-      - ./docker/postgres-data:/var/lib/postgresql/data
+    # volumes:
+    #   - ./docker/postgres-data:/var/lib/postgresql/data
```

Comment:
```suggestion
    - ./docker/postgres-data:/var/lib/postgresql/data
```

Reply:
cant comment these out in the repo - ill show you how to set up an override file that will go in the gitignore

Reply:
Yeah this was an oversight on my part, I never meant to add them in the first place.